_i18n/it/get-started/what-is-monero.md

```diff
@@ -0,0 +1,120 @@
+{% include untranslated.html %}
+<div class="site-wrap">
+    <section class="container">
+            <div class="row">
+                <section class="container about-monero full col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col"><h2>Quello che devi sapere</h2></div>
+                        </div>
+                        <div class="row middle-xs info-block-row private">
+                            <div class="col-lg-7 col-md-6 col-sm-8 col-xs-12  why-text">
+                                <h3>Monero è la cryptomoneta di riferimento tra quelle focalizzate in transazioni private e resistenti alla censura.</h3>
+                                <p>Molte delle cryptomonete esistenti, inclusi Bitcoin ed Ethereum, hanno una blockchain trasparente. Questo significa che le transazioni sono facilmente verificabili e tracciabili da chiunque nel mondo. Inoltre, gli indirizzi di ricezione ed invio per queste transazioni, potrebbero essere collegate all'iudentità reale di una persona. </p>
+                                <p>Monero usa la crittografia per nasconedere l'indirizzo di ricezione ed invio, così come l'ammontare della transazione.</p>
```

Comment:
"Monero usa tecniche all'avanguardia per nascondere i diversi fattori che potrebbero in qualche modo far risalire all'identità del ricevitore e del mittente tra cui gli indirizzi."